packages/experiments/package.json

```diff
@@ -20,9 +20,11 @@
     "code-style": "just-scripts code-style",
     "start": "just-scripts dev",
     "start-test": "just-scripts jest-watch",
-    "update-snapshots": "just-scripts jest -u"
+    "update-snapshots": "just-scripts jest -u",
+    "change": "beachball change"
```

Comment:
You shouldn't need this here. The change command has to be run at the root of the repo now. 

@kenotron Could we add better messaging here, or possibly make the command work from any folder?

Reply:
beachball isn't aware of lerna / monorepo by itself. If we run it here, it'll create the change at this level... the only thing I would suggest maybe to error out here if you did type `yarn change` at this level??

Reply:
Okay I will delete my package.json changes @ecraig12345